packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -348,6 +348,9 @@ class TextStyle extends Diagnosticable {
   /// In [merge], [apply], and [lerp], conflicts between [color] and [foreground]
   /// specification are resolved in [foreground]'s favor - i.e. if [foreground] is
   /// specified in one place, it will dominate [color] in another.
+  ///
```

Comment:
This shouldn't be documented here; the TextStyle class is part of a more fundamental layer than the Material theme or color scheme.

That's my mistake for suggesting that we expand on this point.

Reply:
Sounds good, I'll just remove this piece of documentation then